### DISCUSSION TITLE
Why does imageproxy always give me timeout error?

### DISCUSSION THREAD

**Discussion by davyzhang** [imgproxy/imgproxy#1282] [Q&A]
I tried last version and v3

simply started a docker like this

```
docker pull darthsim/imgproxy:latest
docker run -p 8080:8080 -it darthsim/imgproxy
```
and get the error, I tried arm and amd version and both gave the same error

the image is a google logo which I can download it on the machine with no problem

the url I used to test is:
https://mydoamin/unsafe/resize:fit:150/plain/https://www.google.com/images/branding/googlelogo/2x/googlelogo_color_272x92dp.png

I also have tried with various images, including my own s3 hosted public images, always the same error

```
Started /insecure/rs:fit:300/plain/https://www.google.com/images/branding/googlelogo/2x/googlelogo_color_272x92dp.png  request_id=XEdyquvAiPYh-uhXOGCKq method=GET client_ip=xxx
ERROR   [2024-05-03T02:08:14Z] Completed in 5.004752441s /insecure/rs:fit:300/plain/https://www.google.com/images/branding/googlelogo/2x/googlelogo_color_272x92dp.png  request_id=XEdyquvAiPYh-uhXOGCKq method=GET status=504 client_ip=139.218.20.8 error="Can't download source image: The image request timed out: Get \"https://www.google.com/images/branding/googlelogo/2x/googlelogo_color_272x92dp.png\": context deadline exceeded"
```

**Comment by DarthSim** [comment]
Hey @davyzhang!

This looks like a networking problem in your Docker. Are you able to download the image with a [curl Docker image](https://hub.docker.com/r/curlimages/curl)?
